src/Columns/ColumnMap.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include <Core/Block.h>
+#include <Columns/ColumnArray.h>
+#include <Columns/ColumnVector.h>
+#include <Columns/ColumnTuple.h>
+
+namespace DB
+{
+
+/** Column, that is just group of 2 columns.
```

Comment:
A more verbose description would help, e.g. it's a column of arrays of (key, value) tuples.